File: static_cdn/media_root/submissions/whitehatvicky/FACT.cpp
#include <bits/stdc++.h> 
using namespace std; 
long long int fact(long long int);
int main()
{ 
   long long int cnt = 0,i,n;
    cin>>n;
    n=fact(n);
for (i = 1; i <= sqrt(n); i++) { 
        if (n % i == 0) {
            if (n / i == i)
                cnt++; 
  
            else 
                cnt = cnt + 2; 
        } 
    }
cout<<cnt;

}
long long int fact(long long int n)
{ long long int j;
   for(j=n-1;j>=1;j--)
     n*=j;
return n;
}

File: static_cdn/media_root/submissions/whitehatvicky/FACT.py
import math
n=int(input())
def calc(n):

    if n == 1 or n == 0:
        return 1
    result = 1 
    for x in range(1, n + 1, 1):
        result *= x
    return result
n=calc(n)
cnt=0
for i in range(1,math.ceil(math.sqrt(n))):
    if n%i==0:
        if n//i==i:
            cnt=cnt+1
        else:
            cnt=cnt+2
print(cnt)
  
